controllers/util/k8sutil.go

```diff
@@ -32,8 +32,6 @@ const (
 	// These are the keys in the ocs-operator-config configmap
 	ClusterNameKey              = "CSI_CLUSTER_NAME"
 	RookCurrentNamespaceOnlyKey = "ROOK_CURRENT_NAMESPACE_ONLY"
-	EnableReadAffinityKey       = "CSI_ENABLE_READ_AFFINITY"
-	CephFSKernelMountOptionsKey = "CSI_CEPHFS_KERNEL_MOUNT_OPTIONS"
```

Comment:
are these leftovers of some previous commits?

Reply:
Yup: https://github.com/red-hat-storage/ocs-operator/pull/2294